README.md

```diff
@@ -1,5 +1,8 @@
-# elementary GitHub Actions
+# Elementary GitHub Actions
```

Comment:
`elementary` is always lower-cased, as covered in the [brand resources](https://elementary.io/brand).